package.json

```diff
@@ -9,6 +9,10 @@
     "axios": "^1.3.4",
     "dotenv": "^16.0.3"
   },
+  "scripts": {
+    "prepare:app_env": "cp .env.example .env",
+    "build": "rm -rf ./result.csv && yarn prepare:app_env && node index.js "
```

Comment:
What about  `run`, `execute` or something similar? Since this is not a `build` process.

> yarn prepare:app_env

Furthermore, if we use this approach, the .env file will be overwritten on each execution. Alternatively, we would have to input the actual values into the `.env.example` file. Both scenarios are not ideal.

Reply:
fixup in [4587523](https://github.com/ivan-ha/ua-backup/pull/1/commits/4587523a335250e5b77c696597939e44007f7af0)